src/storyblok/Hero.astro

```diff
@@ -3,15 +3,18 @@ import { storyblokEditable } from "@storyblok/astro";
 
 type Variants = "primary" | "secondary";
 
-const { blok } = Astro.props as { blok: { variant: Variants; text: string } };
+const { blok } = Astro.props as { blok: { variant: Variants; text: string; Text2: string;
+  Image: string; BackgroundColor: string; ButtonText: string;} };
 ---
 
 <div
-  class="bg-cover bg-center"
+  class=`bg-cover bg-center bg-[#63a9dd] relative`
```

Comment:
Why this color is written in line? Is a kind of exception?

Reply:
Yes, it is an exception in this case since we couldn't find a color from the palette that would specifically fit the hero (the closest was tertiary light, but we didn't like how it looked), and we didn't want to add another color since this would be the only case in which it was going to be used.